### PR TITLE
Polymer 3 2019 02 15 081000

### PR DESCRIPTION
**Pull request by ctwomey1**
There's more changes present than in my pull request, but it looks like they are mostly dropping the CRLF for LF.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "lint": "npm run lint:wc && npm run lint:js",
     "lint:js": "eslint . --ext .js,.html test/**/*.js test/**/*.html demo/**/*.js demo/**/*.html",
-    "lint:wc": "polymer lint -i \"./*\" \"test\" \"demo\"",
+    "lint:wc": "polymer lint",
```
Comment (ctwomey1) [inline package.json]: Should i revert this change back to ` "lint:wc": "polymer lint -i \"./*\" \"test\" \"demo\"",`?

Comment (dlockhart) [reply package.json]: Probably it was something that should have been in the patch but was missed.

Comment (svanherk) [reply package.json]: I'll add this to the patch
